src/System.Management.Automation/utils/PsUtils.cs

```diff
@@ -579,15 +190,6 @@ internal static bool IsRunningOnProcessorArchitectureARM()
             {
                 return false;
             }
-#else
-            // Important:
-            // this function has a clone in Workflow.ServiceCore in admin\monad\src\m3p\product\ServiceCore\WorkflowCore\WorkflowRuntimeCompilation.cs
-            // if you are making any changes specific to this function then update the clone as well.
-
-            var sysInfo = new NativeMethods.SYSTEM_INFO();
-            NativeMethods.GetSystemInfo(ref sysInfo);
```

Comment:
It is still used in `internal static ProcessorArchitecture GetProcessorArchitecture(out bool isRunningOnArm)` and then ` internal PSModuleInfo LoadModuleManifest()`.